packages/cli-tools/src/startServerInNewWindow.ts

```diff
@@ -34,7 +36,22 @@ function startServerInNewWindow(
   const packagerEnvFileExportContent = isWindows
     ? `set RCT_METRO_PORT=${port}\nset PROJECT_ROOT=${projectRoot}\nset REACT_NATIVE_PATH=${reactNativePath}`
     : `export RCT_METRO_PORT=${port}\nexport PROJECT_ROOT="${projectRoot}"\nexport REACT_NATIVE_PATH="${reactNativePath}"`;
-  const nodeModulesPath = resolveNodeModuleDir(projectRoot, '.bin');
+  let generatedPath = findPackageDependencyDir('.generated', {
+    startDir: projectRoot,
+  });
+
+  if (!generatedPath) {
+    const newPath = path.join(projectRoot, 'node_modules', '.generated');
+
+    try {
+      await mkdir(newPath);
+    } catch (e) {
+      throw new CLIError(`Failed to create ${newPath}`);
+    }
+
```

Comment:
I don't think we need `fs-extra` here. Node added `recursive` in v10.12.0:
```suggestion
    fs.mkdirSync(newPath, { recursive: true, mode: 0o755 });
```